imod/tests/test_mf6/test_multimodel/test_mf6_partitioning_unstructured.py

```diff
@@ -321,55 +328,120 @@ def test_partitioning_unstructured_with_well(
     simulation.write(original_dir, binary=False)
     simulation.run()
 
-    original_head = simulation.open_head()
-    original_flow_cbc = simulation.open_flow_budget()
+    expected_head = simulation.open_head()
+    expected_flow_cbc = simulation.open_flow_budget()
 
     # Partition the simulation, run it, and save the (merged) results
     split_simulation = simulation.split(partition_array)
 
     split_simulation.write(tmp_path, binary=False)
     split_simulation.run()
 
-    head = split_simulation.open_head()
-    head = head.ugrid.reindex_like(original_head)
+    actual_head = split_simulation.open_head()
+    actual_head = actual_head.ugrid.reindex_like(expected_head)
     # TODO:
     # Uncomment when fixed: https://gitlab.com/deltares/imod/imod-python/-/issues/683
-    cbc = split_simulation.open_flow_budget()
-    cbc = cbc.ugrid.reindex_like(original_flow_cbc)
+    actual_cbc = split_simulation.open_flow_budget()
+    actual_cbc = actual_cbc.ugrid.reindex_like(expected_flow_cbc)
     # Compare the head result of the original simulation with the result of the
     # partitioned simulation.
     np.testing.assert_allclose(
-        head["head"].values, original_head.values, rtol=1e-5, atol=1e-3
+        actual_head["head"].values, expected_head.values, rtol=1e-5, atol=1e-3
     )
     for key in ["flow-lower-face", "flow-horizontal-face"]:
         np.testing.assert_allclose(
-            cbc[key].values, original_flow_cbc[key].values, rtol=1, atol=0.01615156
+            actual_cbc[key].values,
+            expected_flow_cbc[key].values,
+            rtol=1,
+            atol=0.01615156,
         )
 
 
+def run_simulation(tmp_path, simulation, species):
+    simulation.write(tmp_path)
+    simulation.run()
+    head = simulation.open_head()
+    concentration = simulation.open_concentration()
+    flow_budget = simulation.open_flow_budget()
+    flow_budget = flow_budget.sel(time=364)
+    transport_budget = simulation.open_transport_budget(species)
+    transport_budget = transport_budget.sel(time=364)
+    return head, concentration, flow_budget, transport_budget
+
+
+def get_exchange_masks(actual_flow_budget, expected_flow_budget):
+    # create a cell-aray of booleans that is true on the exchange boundary cells and false in other locations
+    is_exchange_cell = (
+        actual_flow_budget["gwf-gwf"]
+        .where(actual_flow_budget["gwf-gwf"] != 0)
+        .notnull()
+    )
```

Comment:
I might be missing something, but this can be simplified to:

```suggestion
    is_exchange_cell = actual_flow_budget["gwf-gwf"] != 0
```